src/Types/ArrayType.php

```diff
@@ -41,8 +41,8 @@ public function convertToPHPValue(mixed $value, AbstractPlatform $platform): mix
 
         $value = is_resource($value) ? stream_get_contents($value) : $value;
 
-        set_error_handler(function (int $code, string $message) use ($value): bool {
-            throw ValueNotConvertible::new($value, $this->getName(), $message);
+        set_error_handler(static function (int $code, string $message) use ($value): bool {
+            throw ValueNotConvertible::new($value, 'array', $message);
```

Comment:
Would it make sense to use the FQN here instead?

Reply:
I don't think it would: we are not converting a string to `ArrayType`, we are converting a `string` to `array` (with `unserialize`)